dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/LoginPropertyTest.java

```diff
@@ -0,0 +1,329 @@
+
+        /*
+         * Make sure the application has come up before proceeding
+         */
+        assertNotNull("Application userRegistry does not appear to have started.",
+                      libertyServer.waitForStringInLog("CWWKZ0001I:.*userRegistry"));
+        assertNotNull("Security service did not report it was ready",
+                      libertyServer.waitForStringInLog("CWWKS0008I"));
+        assertNotNull("Server did not came up",
+                      libertyServer.waitForStringInLog("CWWKF0011I"));
+
+        Log.info(c, "setUp", "Creating servlet connection the server");
+        servlet = new UserRegistryServletConnection(libertyServer.getHostname(), libertyServer.getHttpDefaultPort());
+
+        if (servlet.getRealm() == null) {
+            Thread.sleep(5000);
+            servlet.getRealm();
+        }
+
+        /*
+         * Backup the original server configuration.
+         */
+        originalConfiguration = libertyServer.getServerConfiguration();
+    }
+
+    /**
+     * Configure the embedded LDAP server.
+     *
+     * @throws Exception If the server failed to start for some reason.
+     */
+    private static void setupldapServer() throws Exception {
+        ds = new InMemoryLDAPServer(DN);
+
+        Entry entry = new Entry(DN);
+        entry.addAttribute("objectclass", "top");
+        entry.addAttribute("objectclass", "domain");
+        ds.add(entry);
+
+        /*
+         * Create the user and group.
+         */
+        entry = new Entry(USER_DN);
+        entry.addAttribute("objectclass", "inetorgperson");
+        entry.addAttribute("uid", userUid);
+        entry.addAttribute("sn", userSn);
+        entry.addAttribute("cn", userCn);
+        entry.addAttribute("userPassword", "password");
+        ds.add(entry);
+
+    }
+
+    /**
+     * Convenience method to configure the Liberty server with an {@link LdapRegistry} configuration that
+     * will connect to {@link #ldapServer}.
+     *
+     * @throws Exception If there was an error configuring the server.
+     */
+    private static void updateLibertyServer(boolean addFilters, String... loginProps) throws Exception {
+        ServerConfiguration server = originalConfiguration.clone();
+
+        LdapRegistry ldap = new LdapRegistry();
+
+        ldap.setRealm("LDAPRealm");
+        ldap.setHost("localhost");
+        ldap.setPort(String.valueOf(ds.getLdapPort()));
+        ldap.setBaseDN(DN);
+        ldap.setBindDN(InMemoryLDAPServer.getBindDN());
+        ldap.setBindPassword(InMemoryLDAPServer.getBindPassword());
+        ldap.setLdapType("Custom");
+        ldap.setLdapCache(new LdapCache(new AttributesCache(false, 0, 0, "0s"), new SearchResultsCache(false, 0, 0, "0s")));
+        ldap.setAttributeConfiguration(new AttributeConfiguration());
+        //    ldap.getAttributeConfiguration().getAttributes().add(new Attribute("givenName", "badconfig", "PersonAccount", null, null));
```

Comment:
I see you tested if you could fix it with this... it shouldn't fix it. You should need to create an extendedProperty on PersonAccount of the same name to fix it. Perhaps in the last test where you test a bad property, you should resolve it by adding an extended property.